t/html/pfappserver/Makefile

```
@@ -0,0 +1,69 @@
+#!make
+include .local_env
+export $(shell sed 's/=.*//' .local_env)
+
+BASE_URL ?= https://localhost:1443
+BROWSER ?= chrome
+CONFIG_FILE ?= cypress/config/cypress.config.js
+CONFIG ?= env={}
+#DEBUG ?= cypress:*
+DEBUG ?=
+CI_PIPELINE_ID ?= $$(openssl rand -hex 16)
+PROJECT_ID ?=
+RECORD_KEY ?=
+
+.PHONY: all install test
+
+all:
+	@echo "Usage:"
+	@echo "make install-debian|install-rhel: install libraries"
+	@echo "make install: install cypress"
+	@echo "make test: run tests headless"
+	@echo "make test-e2e: run tests headless, requires PROJECT_ID and RECORD_KEY"
+	@echo "make test-x11: run tests interactively (w/ X11 \$$DISPLAY)"
+	@echo ""
+	@echo "make [target] DEBUG=cypress:* (see https://docs.cypress.io/guides/references/troubleshooting#Log-sources)"
+	@echo "make [target] BROWSER=firefox:nightly"
```

```suggestion
	@echo "make [target] BROWSER= chrome" # or for ex: firefox:nightly
```
To be consistant with default values

---

The default was changed, firefox is preferred due to lower memory usage. It also gives us better coverage that will prevent firefox-only issues in the future.